netsocadmin/templates/page-skeleton.html

```diff
@@ -223,7 +223,9 @@
         <div class="navbar">
             <nav>
                 <div class="nav-wrapper teal lighten-2">
+                    {% if is_logged_in %}
                     <a href="#" data-activates="mobile-demo" class="button-collapse"><i class="material-icons">menu</i></a>
+                    {% endif %}
```

Comment:
Theres 3 links in the sidebar, Website, Tutorials and Blog, that probably would be better of not hidden if not logged in :)

Reply:
Sure, didn't realise they were collapsing too, will rework so we just hide the login dependant options.